app/services/degrees/create_from_dttp.rb

```diff
@@ -22,7 +22,7 @@ def call
     attr_reader :dttp_trainee, :trainee
 
     def create_degrees!
-      dttp_trainee.degree_qualifications.each do |dttp_degree|
+      dttp_trainee.degree_qualifications.not_imported.each do |dttp_degree|
```

Comment:
What is this line doing, where is this defined?

Reply:
That's a scope provided by the [AR Enum module](https://edgeapi.rubyonrails.org/classes/ActiveRecord/Enum.html), since we have `state` as an enum, we get scopes for each of the enums for free, i.e, `DegreeQualification.imported` and `DegreeQualification.not_imported` and so on